packages/launchpad/cypress/e2e/project-setup.cy.ts

```diff
@@ -815,4 +815,22 @@ describe('Launchpad: Setup Project', () => {
       cy.get('code').should('contain.text', 'npm install -D ')
     })
   })
+
+  describe('openLink', () => {
+    it('opens docs link in the default browser', () => {
+      scaffoldAndOpenProject('pristine-with-e2e-testing')
+
+      cy.visitLaunchpad()
+
+      cy.get('[data-cy-testingtype="component"]').click()
+      cy.get('[data-testid="select-framework"]').click()
+      cy.findByText('Nuxt.js (v2)').click()
+      cy.findByText('Next Step').click()
+      fakeInstalledDeps()
+      cy.findByRole('button', { name: 'Continue' }).click()
+      cy.intercept('POST', 'mutation-ExternalLink_OpenExternal').as('OpenExternal')
+      cy.findByText('Learn more.').click()
+      cy.wait('@OpenExternal')
```

Comment:
Should we assert the correct arg was included in the request?

Reply:
Edit: probably irrelevant if you make the change mentioned by @tbiethman.

Reply:
Added in commit 15206c91e44ea883215b0a5c4d0f5f736734ff7e